### DISCUSSION TITLE
Github Pages 100GB Limits

### DISCUSSION THREAD

**Discussion by kamechan8** [community/community#22155] [Other Feature Feedback, Questions, & Ideas]
<p>I plan to use Github Pages for the event website.<br>
It’s expected to get an enormous access in a few days.</p>
<p>Its <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages" rel="noopener nofollow ugc">document</a> says</p>
<blockquote>
<p>GitHub Pages sites have a soft bandwidth limit of 100GB per month.</p>
</blockquote>
<p>but I can’t see limit for the number of access or daily limit.</p>
<p>Is “100GB/month” an only limit?</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Yes. Per the docs you linked above: <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages#guidelines-for-using-github-pages" class="inline-onebox" rel="noopener nofollow ugc">About GitHub Pages - GitHub Docs</a> there is a soft limit which if exceeded might trigger GitHub to take action in various ways but there’s no other limits other than what is stated in the docs.</p>
